docs/Testing.md

```diff
@@ -1,6 +1,71 @@
 # Boss Testing Guide
 This document covers running unit tests and integration tests within an existing Bosslet.
 
+
+### Copy up dynamo information in spdb for testing
+Now the spdb is a pypi installable it doesn't include the contents of 
+spdb/spdb/spatialdb/dynamo/
+You need to tar up the contents of that directory and use ssh-tunnel to copy it to the endpoint for testing
+```bash
+$ cd spdb/spdb/spatialdb
+$ tar czf /path/to/spdb-spatialdb-dynamo.tar.gz ./dynamo
+$ cd spdb
+$ tar rzf /path/to/spdb-spatialdb-dynamo.tar.gz ./requirements-test.txt
```

Comment:
Probably easier to just `pip install -r requirements-text.txt` on the endpoint?

Reply:
You mean run that as part of the packer script?  I like that idea, if that is what you mean.